code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -259,10 +256,13 @@
 	else if(user.bloodcrawl == BLOODCRAWL)
 		to_chat(user, "You feel diffr-<span class = 'danger'> CONSUME THEM! </span>")
 		user.bloodcrawl = BLOODCRAWL_EAT
-	else
+	else if(user.bloodcrawl == BLOODCRAWL_EAT)
```

Comment:
The if check is not needed here. There are only three options.
1. bloodcrawl == 0
2. bloodcrawl == 1 (`BLOODCRAWL`)
3. bloodcrawl == 2 (`BLOODCRAWL_EAT`)

If bloodcrawl does not equal 0 or 1, the only other option is 2, so a simple `else` works fine here.

Reply:
It makes it easier to understand imo, also else should be for 0 since else is usually called if things are false and 0 is attributed to things being false

Reply:
Actually yeah, I like the clarity of having this here, so ignore what I said.